src/content/2/en/part2d.md

```diff
@@ -594,7 +594,7 @@ export default { getAll, create, update } // highlight-line
 ```
 
 
-In this more compact definition we are using a [new feature](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Property_definitions) that was added to JavaScript in the ES6 language specification. The feature specifically enables this more compact way of defining objects.
+In defining the object using this shorter notation, we make use of a [new feature](https://developer.mozilla.org/enUS/docs/Web/JavaScript/Reference/Operators/Object_initializer#Property_definitions) that was introduced to JavaScript through ES6, enabling a slightly more compact way of defining objects using variables.
```

Comment:
```suggestion
In defining the object using this shorter notation, we make use of a [new feature](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Property_definitions) that was introduced to JavaScript through ES6, enabling a slightly more compact way of defining objects using variables.
```

Reply:
Please commit this typo fix before merging.